src/components/styleElements/buttons/ShotsButton/ShotsButton.css

```diff
@@ -0,0 +1,15 @@
+.shotsButton {
```

Comment:
I stated creating the red and green buttons that will show up on the "Shots" page